server-launcher/build.gradle

```diff
@@ -165,15 +165,14 @@ task defaultCommandSnippetsZip(type: Zip, dependsOn: [':server-launcher:cloneCom
 task createVersionFile {
   def file = file("${project.buildDir}/version.txt")
   inputs.property('version', rootProject.version)
-  inputs.property('distVersion', rootProject.distVersion)
   inputs.property('gitRevision', rootProject.gitRevision)
 
   outputs.file file
 
   doLast {
     file.getParentFile().mkdirs()
     file.withWriter { out ->
-      out.write("${rootProject.version}(${rootProject.distVersion}-${rootProject.gitRevision})")
+      out.write("${rootProject.version} (${rootProject.gitRevision})")
```

Comment:
what backups @ketan? Did you mean `BackupService`? If not, could you point it out?